### DISCUSSION TITLE
Key for changing tab does not get remapped

### DISCUSSION THREAD

**Discussion by ajojoxiii** [sumatrapdfreader/sumatrapdf#3521] [Q&A]
In Sumatra version 3.4.6 I'm trying to remap NextTab/PrevTab to more familiar to me Ctrl+PgDn and Ctrl+PgUp but it doesn't work for me

```
Shortcuts [
	[
		Cmd = CmdNextTab
		Key = Ctrl + PgDown
	]
	[
		Cmd = CmdPrevTab
		Key = Ctrl + PgUp
	]
]

```

What am I doing wrong?

**Comment by GitHubRulesOK** [comment]
double check spelling is it currently the default
Ctrl + page up is back and page down is forward
 so to reverse I need a more intuitive
```
Shortcuts [
	[
		Cmd = CmdNextTab
		Key = Ctrl + PageUp
	]
	[
		Cmd = CmdPrevTab
		Key = Ctrl + PageDown
	]
]
```

**Reply by ajojoxiii** [reply]
I copied the text you provided and there was no change in the behavior.

**Reply by GitHubRulesOK** [reply]
that combination works for me in 3.5.#### but not 3.4.6 (2022-06-08) as its a newer pre-release addition (thus confusingly was not added to main release even though added earlier Apr 13, 2022) along with many others. https://github.com/sumatrapdfreader/sumatrapdf/issues/2482#issuecomment-1098267293
